templates/single-tapestry.php

```diff
@@ -29,9 +29,11 @@ function addNonceToScript()
 
 get_header(); ?>
 
+<?php require "utility.php";?>
 <div id="primary" class="content-area col-md-12">
     <main id="main" class="post-wrap" role="main">
-        <?php while (have_posts()) : the_post(); ?>
+            <?php while ( have_posts() ) : the_post(); ?>
```

Comment:
This is too indented. Indentation should be 4 spaces all the time.

Reply:
fixed